src/Workload/Microsoft.NET.Sdk.Maui.Manifest/Microsoft.NET.Sdk.Maui.Manifest.csproj

```diff
@@ -118,22 +118,14 @@
   </Target>
 
   <Target Name="_CopyAdditionalFIles" AfterTargets="Build">
-    <Copy SourceFiles="$(OutputPath)Rollback.json" DestinationFiles="$(PackageOutputPath)/metadata/rollbacks/$(Version)/$(SourceRevisionId).json" />
-    <Copy SourceFiles="$(OutputPath)Rollback.json" DestinationFiles="$(PackageOutputPath)/metadata/rollbacks/$(Version)/$(PackageVersion).$(BUILDVERSION).json" />
-    <Copy SourceFiles="$(OutputPath)Rollback.json" DestinationFiles="$(PackageOutputPath)/metadata/rollbacks/$(Version)/$(PackageVersion).json" />
-    <Copy SourceFiles="$(OutputPath)Rollback.json" DestinationFiles="$(PackageOutputPath)/metadata/rollbacks/$(Version)/$(PackageReferenceVersion).json" />
-    <Copy SourceFiles="$(OutputPath)Rollback.json" DestinationFiles="$(PackageOutputPath)/metadata/rollbacks/$(SourceRevisionId).json" />
-    <Copy SourceFiles="$(OutputPath)Rollback.json" DestinationFiles="$(PackageOutputPath)/metadata/rollbacks/$(PackageVersion).json" />
-    <Copy Condition="$(CI) and '$(BUILD_SOURCEBRANCHNAME)' != ''" SourceFiles="$(OutputPath)Rollback.json" DestinationFiles="$(PackageOutputPath)/metadata/rollbacks/$(BUILD_SOURCEBRANCHNAME).json" />
+    <Copy SourceFiles="$(OutputPath)Rollback.json" DestinationFiles="$(PackageOutputPath)/metadata/rollbacks/$(SourceRevisionId)_rollback.json" />
```

Comment:
We don t need so many copies of these files, but we need to suffix the filename as they are pushed on a flat container